src/react-imgix.js

```diff
@@ -80,15 +102,35 @@ function buildSrc({
       const dpr3 = constructUrl(rawSrc, { ...srcOptions, dpr: 3 });
       srcSet = `${dpr2} 2x, ${dpr3} 3x`;
     } else {
+      let showARWarning = false;
       const buildSrcSetPair = targetWidth => {
-        const url = constructUrl(rawSrc, {
+        let urlParams = {
           ...srcOptions,
           width: targetWidth
-        });
+        };
+        const aspectRatioDecimal = aspectRatio && parseAspectRatio(aspectRatio);
+        if (aspectRatio != null && aspectRatioDecimal === false) {
```

Comment:
See above. 
If all type checks just happen inside `parseAspectRatio`, `aspectRatio != null` can be removed.

Reply:
The reason for having `aspectRatio != null` here is so that a warning is only thrown when the user has tried to pass something. e.g. it wouldn't throw a warning when no prop is passed, but it would when the user passes "".

Reply:
Good call! Totally makes sense to check if the prop is there, before showing a warning.
One questions I guess is wouldn't it be equal to `undefined` instead of `null`, if the prop is absent/not passed?

Reply:
Definitely! This is the reason for using `!=` rather than `!==`, as this covers both `undefined` and `null`.

i.e.
```js
undefined != null // false
undefined !== null // true
```

Reply:
Totally! Great work on PR, LGTM 👍

Reply:
Thanks @maxkolyanov! 🙌 thanks again for your work for the base for this PR.